model/Core/Vocabularies/RelationshipType.md

```diff
@@ -98,3 +98,5 @@ name completes the sentence:
 - underInvestigationFor: The `from` Vulnerability impact is being investigated for each `to` Element. The use of the `underInvestigationFor` type is constrained to `VexUnderInvestigationVulnAssessmentRelationship` classed relationships.
 - usesTool: The `from` Element uses each `to` Element as a tool, during a LifecycleScopeType period.
 - validatedOn: The `from` Element has been validated on the `to` Element(s).
+- isVerifiedBy: The `from` RequirementVerification is a verifies process of `to` Requirement.
```

Comment:
```suggestion
- verifiedBy: The `from` RequirementVerification is a verification process of `to` Requirement.
```

Follows the pattern of invokedBy, modifiedBy, packagedBy, patchedBy, reportedBy, ..., omitting "is" prefix for brevity.